samples/pom.xml

```diff
@@ -34,6 +34,11 @@
 
   <name>Apache Apex Malhar (incubating) Samples</name>
 
+  <properties>
+    <semver.plugin.skip>true</semver.plugin.skip>
```

Comment:
Is semver plugin enabled for samples? japicmp is part of plugin management and should not be invoked unless explicitly enabled.